scripts/app.js

```diff
@@ -0,0 +1,187 @@
+        case 'selectHigh':
+            pledgesRadioBtns[0].checked = false;
+            pledgesRadioBtns[1].checked = false;
+            pledgesRadioBtns[2].checked = false;
+            pledgesRadioBtns[3].checked = true;
+            break;
+    }
+}
+function highlightPledge() {
+    pledgeTexts.forEach((text, i) => {
+        text.addEventListener('click', (e) => {
+            if (modalItem[i].classList[1] === 'disabled') {
+                return;
+            }
+            if (e.target.classList[0]) {
+                pledgesRadioBtns[i].checked = true;
+            }
+        });
+    });
+    pledgesRadioBtns.forEach((btn, i) => {
+        if (btn.checked) {
+            enterPledge[i].classList.remove('enterPledge');
+            modalItem[i].classList.add('picked');
+        } else {
+            enterPledge[i].classList.add('enterPledge');
+            modalItem[i].classList.remove('picked');
+        }
+    });
+}
+function toggleNav(value) {
+    switch (value) {
+        case 'hamburger':
+            navHamburger.classList.remove('hamburger');
+            navHamburger.classList.add('close');
+            navHamburger.src = closeIcon;
+            navOverlay.classList.add('linearBackground');
+            nav.classList.remove('hide');
+            body.classList.add('bodyOverflow');
+            break;
+        case 'close':
+        case 'logo':
+        case 'navOverlay':
+        case 'closeNav':
+        case 'getStarted':
+            navHamburger.classList.add('hamburger');
+            navHamburger.classList.remove('close');
+            navHamburger.src = hamburgerIcon;
+            navOverlay.classList.remove('linearBackground');
+            nav.classList.add('hide');
+            body.classList.remove('bodyOverflow');
+            break;
+    }
+}
+function toggleModal(value) {
+    switch (value) {
+        case 'backProjectBtn':
+        case 'selectMed':
+        case 'selectMin':
+        case 'selectHigh':
+        case 'getStarted':
+            overlay.classList.remove('hidden');
+            modal.classList.remove('hidden');
+            body.classList.add('bodyOverflow');
+            break;
+        case 'closeModal':
+        case 'overlay':
+            overlay.classList.add('hidden');
+            modal.classList.add('hidden');
+            body.classList.remove('bodyOverflow');
+            break;
+    }
+}
+function thankYouModal(innerText) {
+    if (innerText === 'Continue') {
+        numBackers++;
+        thankYou.classList.remove('hidden');
+        overlay.classList.remove('hidden');
+        modal.classList.add('hidden');
+        body.classList.add('bodyOverflow');
+        backers.innerText = +numBackers;
+        let newAmount;
+        if (pledgesRadioBtns[0].checked) {
+            newAmount = +toNumber(currentAmount) + +amount0.value;
+            currentAmount.innerText = `$${newAmount.toLocaleString()}`;
+            backedAmount(currentAmount, maxAmount);
+        }
```

Comment:
```suggestion
        
        if (pledgesRadioBtns[0].checked) {
            newAmount = +toNumber(currentAmount) + +amount0.value;
            currentAmount.innerText = `$${newAmount.toLocaleString()}`;
            backedAmount(currentAmount, maxAmount);
        }
        
```
uvek ostavi po prazan red ispred i iza if bloka - zbog preglednosti